diff --git a/chrome/browser/resources/tools/optimize_webui.gni b/chrome/browser/resources/tools/optimize_webui.gni
index f21fbd980f438872099497c6910d964c0cb57379..500cfe4921197cca006933508cdd79a5f7dd9dde 100644
--- a/chrome/browser/resources/tools/optimize_webui.gni
+++ b/chrome/browser/resources/tools/optimize_webui.gni
@@ -48,6 +48,7 @@ template("optimize_webui") {
       "chrome://resources/polymer/v3_0/|$polymer_path",
       "chrome://resources/|$resources_path",
     ]
+    import("//adrbrowsiel/browser/resources/tools/adrbrowsiel_optimize_webui.gni") deps += adrbrowsiel_webui_deps external_paths += adrbrowsiel_webui_external_paths
     if (defined(invoker.external_paths)) {
       external_paths += invoker.external_paths
     }
